README.md

```diff
@@ -111,6 +111,12 @@ plaidClient.getIncome(access_token, cb);
 // getCreditDetails(String, Function)
 plaidClient.getCreditDetails(access_token, cb);
 
+// getHoldings(String, Function)
+plaidClient.getHoldings(access_token, cb);
+// getInvestmentTransactions(String, Date(YYYY-MM-DD), Date(YYYY-MM-DD),
+// Object?, Function)
+plaidClient.getInvestmentTransactions(access_token, cb);
```

Comment:
Are params missing here. Looking at the equivalent python function it seems so:
https://github.com/plaid/plaid-python/pull/169/files#diff-469bbc1fb03163938e0174980a8059c5R7

Reply:
good call out! updated